vita3k/gui/src/settings_dialog.cpp

```diff
@@ -407,7 +407,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             }
             ImGui::SameLine();
         } else {
-            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the last firmware.");
+            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the firmware.");
```

Comment:
wrong, is last firmwre

Reply:
##### could have various correct sentences like
* install the latest firmware
* install the PSVita firmware
* install firmware(?
##### im not sure about "install the last firmware" as it doesn't sound right